guides/common/modules/con_prerequisites-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,45 @@
+[id="prerequisites-for-{project-context}-on-aws"]
+= Prerequisites for {Project} on AWS
+
+Installing and running {ProjectServer} and {SmartProxyServers} on Amazon Web Services (AWS) has additional requirements to your environment.
+
+* Ensure that the Amazon EC2 instance type meets or exceeds the {InstallingServerDocURL}system-requirements_{project-context}[System requirements] in _{InstallingServerDocTitle}_.
+For the best performance, use an https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-optimized-instances.html[AWS storage optimized instance].
+ifndef::foreman-deb[]
+* Use {InstallingServerDocURL}satellite-storage-requirements_{project-context}[Storage requirements] in _{InstallingServerDocTitle}_ to understand and assign the correct storage to your AWS EBS volumes.
+* Store content on an EBS volume that is separate to the boot volume.
+* Mount your content EBS volume separately in the operating system.
+* Optional: Store other data on a separate EBS volume.
+endif::[]
+* If you want {ProjectServer} and {SmartProxyServer} to communicate using external DNS hostnames, open the required ports for communication in the AWS Security Group that is associated with the instance.
+
+.Amazon Web Service requirements
+ifdef::foreman-deb[]
+* Creating and accessing {DL} images in AWS
+endif::[]
+ifndef::foreman-deb[]
+* Creating and accessing {EL} images in AWS
+endif::[]
+* Editing network access in AWS Security
+* Creating EC2 instances and EBS volumes
+* Launching EC2 instances
+* Importing and exporting virtual machines in AWS
+* Using AWS Direct Connect
+
+.System requirements
+* To install {ProjectServer} in your AWS environment, ensure that it meets {ProjectServer} system requirements.
+For more information, see {InstallingServerDocURL}system-requirements_{project-context}[System requirements] in _{InstallingServerDocTitle}_.
+* To install {SmartProxyServer} in your AWS environment, ensure that it meets {SmartProxyServer} system requirements.
+For more information, see {InstallingSmartProxyDocURL}system-requirements_{smart-proxy-context}[System requirements] in _{InstallingSmartProxyDocTitle}_.
```

Comment:
This links to the Storage and System prereqs again. See my point above about perhaps finding a way to link to them only once.